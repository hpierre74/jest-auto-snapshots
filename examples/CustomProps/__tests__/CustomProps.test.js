import snap from 'jest-auto-snapshots';
import CustomProps from '../CustomProps';

snap(CustomProps, '../CustomProps.jsx', { fixturesByPropType: { customPropType: 'customproptest' } });
