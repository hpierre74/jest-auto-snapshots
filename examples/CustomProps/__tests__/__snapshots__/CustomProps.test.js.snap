// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > CustomProps Matches snapshot when passed all props: when passed all props 1`] = `<div />`;
